StackExchange.Redis/StackExchange/Redis/GeoEntry.cs

```diff
@@ -0,0 +1,144 @@
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+
+namespace StackExchange.Redis
+{
+
+    /// <summary>
+    /// Describes a GeoEntry element with the corresponding value
+    /// GeoEntries are stored in redis as SortedSetEntries
+    /// </summary>
+    public struct GeoEntry : IEquatable<GeoEntry>, IEquatable<SortedSetEntry>, IComparable, IComparable<GeoEntry>, IComparable<SortedSetEntry>
+    {
+        internal readonly RedisValue element;
```

Comment:
haven't checked what uses these yet, but do they need to be `internal`? `private` is a good default - if not, it would be good to state why not